resources/aws_ec2.go

```diff
@@ -66,17 +87,31 @@ var AwsRegions = []string{
 // AWS credentials must be present in ~/.aws - For detailed instructions see
 // http://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html
 type AwsEc2Res struct {
-	BaseRes `yaml:",inline"`
-	State   string `yaml:"state"`   // state: running, stopped, terminated
-	Region  string `yaml:"region"`  // region must match an element of AwsRegions
-	Type    string `yaml:"type"`    // type of ec2 instance, ie. t2.micro
-	ImageID string `yaml:"imageid"` // imageid must be available on the chosen region
+	BaseRes     `yaml:",inline"`
+	State       string `yaml:"state"`       // state: running, stopped, terminated
+	Region      string `yaml:"region"`      // region must match an element of AwsRegions
+	Type        string `yaml:"type"`        // type of ec2 instance, ie. t2.micro
+	ImageID     string `yaml:"imageid"`     // imageid must be available on the chosen region
+	SnsEndpoint string `yaml:"snsendpoint"` // the public url endpoint to subscribe to events, ie. http://server:12345/
```

Comment:
While it is entirely correct that this is an SnsEndpoint, this is *our* public API that we're building. So if we can abstract away the weird names of things that amazon invents and instead present more logical interfaces, this is a good idea IMO. For example, we might instead want to call this `WatchEndpoint`, `WebHookEndpoint` or something else clever. I need to think on it, but maybe you have an idea of something that doesn't include "SNS".